src/IQFeed.CSharpApiClient/Lookup/Historical/Messages/TickMessage.cs

```diff
@@ -97,43 +119,21 @@ public static bool TryParse(string message, out TickMessage tickMessage)
             int dayCode = default;
 
             var values = message.SplitFeedMessage();
-            if (values.Length <= 11)
-            {
-                // old protocol < 6.1 data
-                var parsed = values.Length >= 10 &&
-                             DateTime.TryParseExact(values[0], TickDateTimeFormat, CultureInfo.InvariantCulture,
-                                 DateTimeStyles.None, out timestamp) &&
-                             double.TryParse(values[1], NumberStyles.Any, CultureInfo.InvariantCulture, out last) &&
-                             int.TryParse(values[2], NumberStyles.Any, CultureInfo.InvariantCulture, out lastSize) &&
-                             int.TryParse(values[3], NumberStyles.Any, CultureInfo.InvariantCulture, out totalVolume) &&
-                             double.TryParse(values[4], NumberStyles.Any, CultureInfo.InvariantCulture, out bid) &&
-                             double.TryParse(values[5], NumberStyles.Any, CultureInfo.InvariantCulture, out ask) &&
-                             long.TryParse(values[6], NumberStyles.Any, CultureInfo.InvariantCulture, out tickId) &&
-                             char.TryParse(values[7], out basisForLast) &&
-                             int.TryParse(values[8], NumberStyles.Any, CultureInfo.InvariantCulture,
-                                 out tradeMarketCenter);
-
-                if (!parsed)
-                    return false;
-            }
-            else
-            {
-                var parsed = values.Length >= 10 &&
-                             DateTime.TryParseExact(values[0], TickDateTimeFormat, CultureInfo.InvariantCulture, DateTimeStyles.None, out timestamp) &&
-                             double.TryParse(values[1], NumberStyles.Any, CultureInfo.InvariantCulture, out last) &&
-                             int.TryParse(values[2], NumberStyles.Any, CultureInfo.InvariantCulture, out lastSize) &&
-                             int.TryParse(values[3], NumberStyles.Any, CultureInfo.InvariantCulture, out totalVolume) &&
-                             double.TryParse(values[4], NumberStyles.Any, CultureInfo.InvariantCulture, out bid) &&
-                             double.TryParse(values[5], NumberStyles.Any, CultureInfo.InvariantCulture, out ask) &&
-                             long.TryParse(values[6], NumberStyles.Any, CultureInfo.InvariantCulture, out tickId) &&
-                             char.TryParse(values[7], out basisForLast) &&
-                             int.TryParse(values[8], NumberStyles.Any, CultureInfo.InvariantCulture, out tradeMarketCenter) &&
-                             int.TryParse(values[10], NumberStyles.Any, CultureInfo.InvariantCulture, out tradeAggressor) &&
-                             int.TryParse(values[11], NumberStyles.Any, CultureInfo.InvariantCulture, out dayCode);
-
-                if (!parsed)
-                    return false;
-            }
+            var parsed = values.Length >= 10 &&
```

Comment:
you changed the previous constant to TickCsvLengthProtocol60 so this should be likewise